src/mscorlib/src/System/Runtime/Intrinsics/X86/Aes.cs

```diff
@@ -13,6 +14,7 @@ namespace System.Runtime.Intrinsics.X86
     [CLSCompliant(false)]
     public static class Aes 
     {
+        [Intrinsic]
```

Comment:
Since every method in this namespace will be marked as intrinsic eventually, I am wondering whether we should rather special case `System.Runtime.Intrinsics.*` (here)[https://github.com/dotnet/coreclr/blob/master/src/vm/methodtablebuilder.cpp#L5086] and set the flag for all methods under this namespaces.


https://github.com/dotnet/coreclr/blob/master/src/vm/methodtablebuilder.cpp#L5088

Reply:
> we should rather special case `System.Runtime.Intrinsics.*` ... and set the flag for all methods under this namespaces.

Doesn't this reduce intrinsic recognition overhead?

Reply:
Well, each `[Intrinsic]` adds size to `System.Private.CoreLib`. It is hard to tell what is faster, and it is not that important because the result is cached. Special casing the namespace will be definitely smaller (rought estimate 5-10 kB of disk footprint).

Reply:
I see. Could we change `IntrinsicsAttribute` to be able to apply on `Class` and `Struct` (or larger scopes)? Then specially treat `System.Runtime.Intrinsics.*`?

Reply:
You cannot apply the attribute on large scopes. It would have to be on each class or struct. It does not look like a particularly great option to me.

Reply:
> It does not look like a particularly great option to me.

How about considering this optimization later (in a new issue)? Or you prefer to address it in this PR?

Reply:
If we do it now, it would save you from editing like dozen of .cs files. But if you prefer to do it separately - it is fine with me.

Reply:
> If we do it now, it would save you from editing like dozen of .cs files. 

Good point 😄. But let's consider the "optional expand" issue at first, that is more important.